autosklearn/util/backend.py

```diff
@@ -33,20 +35,34 @@ def __init__(self,
                  temporary_directory,
                  output_directory,
                  delete_tmp_folder_after_terminate,
-                 delete_output_folder_after_terminate):
-        self._prepare_directories(temporary_directory, output_directory)
+                 delete_output_folder_after_terminate,
+                 shared_mode=False):
+
+        # Check that the names of tmp_dir and output_dir is not the same.
+        if temporary_directory == output_directory \
+            and temporary_directory is not None:
+            raise ValueError("The names of tmp dir and output dir "
```

Comment:
Could you please change this to `The temporary and the output directory must be different`?